<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVisualizationSegments</class>
 <widget class="QDialog" name="DialogVisualizationSegments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization of experimental segments</string>
  </property>
  <widget class="QLabel" name="lPanelVisualization">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1021</width>
     <height>640</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="rad_smooth">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>270</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtered data</string>
   </property>
  </widget>
  <widget class="QLabel" name="lLabel_15">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>650</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fixations</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rad_rough">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>240</y>
     <width>102</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rough data</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPanelXY_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>1021</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbPupilDilation">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>340</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pupil Dilation</string>
   </property>
  </widget>
  <widget class="QLabel" name="lLabel_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPanelXY">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>1021</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>305</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Additional options:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lLabel_16">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>700</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gaze Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="lsecsSegment">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>700</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPanelMissingData">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>910</y>
     <width>1021</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbFixations">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>380</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Paint fixations</string>
   </property>
  </widget>
  <widget class="QLabel" name="lProcess">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>350</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>210</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Data type:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_play">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_cancel">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPanelFixations">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>1021</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Segment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Fragment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>110</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select a background image:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_prevSegment">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_nextSegment">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lSegment">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lFragment">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_prevFragment">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_nextFragment">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>180</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>* It only accepts static images</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_inputImage">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>140</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_selectImage">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>140</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select image</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPanelBackground">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1021</width>
     <height>640</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lPanelBackground</zorder>
  <zorder>lPanelVisualization</zorder>
  <zorder>rad_smooth</zorder>
  <zorder>lLabel_15</zorder>
  <zorder>rad_rough</zorder>
  <zorder>cbPupilDilation</zorder>
  <zorder>lLabel_14</zorder>
  <zorder>lPanelXY</zorder>
  <zorder>label_2</zorder>
  <zorder>lLabel_16</zorder>
  <zorder>lsecsSegment</zorder>
  <zorder>lPanelMissingData</zorder>
  <zorder>cbFixations</zorder>
  <zorder>lProcess</zorder>
  <zorder>label</zorder>
  <zorder>b_play</zorder>
  <zorder>b_cancel</zorder>
  <zorder>lPanelFixations</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>b_prevSegment</zorder>
  <zorder>b_nextSegment</zorder>
  <zorder>lSegment</zorder>
  <zorder>lFragment</zorder>
  <zorder>b_prevFragment</zorder>
  <zorder>b_nextFragment</zorder>
  <zorder>label_6</zorder>
  <zorder>b_inputImage</zorder>
  <zorder>l_selectImage</zorder>
  <zorder>lPanelXY_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
